modules/sdpolicy-am-gcp-compute-types.adoc

```diff
@@ -8,7 +8,7 @@
 {product-title} offers the following worker node types and sizes on Google Cloud that are chosen to have a common CPU and memory capacity that are the same as other cloud instance types:
 [NOTE]
 ====
-`e2` and `a2` compute types are available for CCS only.
+`e2`, `a2` and `a3` compute types are available for CCS only.
```

Comment:
IBM Style Guide advises using a comma to separate items in a series of three or more (https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series. 
```suggestion
`e2`, `a2`, and `a3` compute types are available for CCS only.
```